### ISSUE TITLE
Web uploader doesn't work when trying to upload too many files

### ISSUE THREAD

**Issue by poma** [poma/hotsapi#20]
https://www.reddit.com/r/heroesofthestorm/comments/6xsted/hotsapinet_first_results_and_logo_contest/dmigjw1/

>I wish the web uploader allowed for any numbers of replays like it does on hotslogs. Now it is just a guessing game trying to upload stuff. "How many should I try to select". If you select too many, nothing happens.

I've not verified this bug yet.

**Comment by FrancoisWiid** [comment]
Seems to be at 500 when it doesn't upload.

**Comment by ekzor** [comment]
Tried to recreate this bug in Firefox 55 with both a local instance and the live site and had no trouble getting it to start uploading with 1700 replays selected.

@FrancoisWiid What browser are you using?

**Comment by ekzor** [comment]
Bug recreated in Chrome 60. The exact number of replays required to trigger the bug is variable and appears to be based on the length of the string of filenames the user attempts to upload.

I managed to recreate this same upload bug using a second popular upload plug-in (dropzone.js) and the number of files i managed to upload was exactly the same, again only in Chrome, so this is not a problem with the hotsapi.net site and it's not one I'm sure we can code our way around.

I tested this with the following method:

1. copied a list of all replay filenames on my system into Excel
2. sorted them the same way as in Chrome
3. Pinpointed how many files were selected when the bug would occur:
a. Starting from 'Battlefield of Eternity', I got to 975 files before failure
b. Starting from 'Blackheart's Bay', I got to 1011 files before failure

Interesting sidenote: if you concatenate all the filenames using a single character, it looks like Chrome maxes out just over 32600 chars, or just under 2^15 for the filename string.

Unfortunately there's no way to code around this since the limitation is in Chrome. It's possible that other browsers have this issue too, and it's possible that other versions of Chrome do not. The best thing we can do is notify Chrome users that if they wish to upload more than ~700 replays (based on the longest map name, tomb of the spider queen), they should either use the uploader app, use Firefox, or break their uploads into batches of 600~700 each.
